env_example

```diff
@@ -0,0 +1,15 @@
+# Non privileged user credentials (to execute commands with)
+POSTGRES_USERNAME=user
+POSTGRES_PASSWORD=password
+
+# In-app admin credentials
+APP_ADMINUSER=admin
```

Comment:
Please, separate words using "_"

Reply:
Changed to more readable variable names.